env.example

```diff
@@ -56,7 +56,7 @@ MATTERMOST_CLIENT_PLUGINS_PATH=./volumes/app/mattermost/client/plugins
 
 ## This will be 'mattermost-enterprise-edition' or 'mattermost-team-edition' based on the version of Mattermost you're installing.
 MATTERMOST_IMAGE=mattermost-enterprise-edition
-MATTERMOST_IMAGE_TAG=5.39
+MATTERMOST_IMAGE_TAG=6.0.2
```

Comment:
Sorry for the delay...Can you please bump it to **6.2**?

Reply:
My Trial system currently runs on 6.0.4 > so far so good

Reply:
I'm on 6.0.4 so far all good

Reply:
This could be increased to **6.3** by now.

As this is a chore, eventually this could possibly be automatised with a hook to the Docker builds? https://hub.docker.com/r/mattermost/mattermost-team-edition/tags

---

Which brings me to the slightly off topic question, why the digest sha hashes for tags like `6.3` and `release-6.3` differ from each other. Would anybody here know which one is better to use? Else I could raise a new issue.

Reply:
6.4 is out.

Reply:
I'm updated from 5.39 to 6.4.1 without any issues.

Reply:
let's add the latest 6.4.1

Reply:
It's `6.5.0` now. Is there a chance to get this automated? Or could we elect someone from the community to maintain this version?